docs/content/users/configuration/config.md

```diff
@@ -103,6 +103,16 @@ Extra Debian packages for the project’s database container.
 
 Example: `dbimage_extra_packages: ["less"]` will add the `less` package when the database container is built.
 
+## `ddev_version_constraint`
+
+You can configure a [ddev version constraint](https://github.com/Masterminds/semver#checking-version-constraints) that will be validated against the running ddev and prevent any command from running if it doesn't match.
+
+Note: this will only work with ddev versions above v1.22.4, older ddev versions will ignore this setting.
```

Comment:
Drop “Note:” which isn’t adding anything—or use an admonition if this needs urgent visual emphasis. Also capitalize DDEV again, fix punctuation, and remove redundant “DDEV”:

```suggestion
This will only work with DDEV versions above v1.22.4; older versions will ignore this setting.
```